pymc/tests/test_variational_inference.py

```diff
@@ -9,7 +9,7 @@
 #   Unless required by applicable law or agreed to in writing, software
 #   distributed under the License is distributed on an "AS IS" BASIS,
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
```

Comment:
why changing this line?

Reply:
This is a mistake, and I will correct it.

Reply:
This is a mistake, and I will correct it.